onboarding_v2/next_steps.md

```diff
@@ -1,16 +1,16 @@
 <div class="container-fluid">
 
 <p>To experiment with sample data, use our <a href="../dashboard/tutorial-intro"><strong>tutorial integration</strong></a>.</p>
-<p><a href="../dashboard/tutorial-intro"><img src="images/tutorial_integration_pic.png"></img> </a></p>
+<p><a href="../dashboard/tutorial-intro" aria-label="Tutorial integration"><img src="images/tutorial_integration_pic.png" alt="Tutorial integration"></img> </a></p>
 <p>&nbsp;</p>
 <p><strong>Invite your co-workers to collaborate in Wavefront!</strong></p>
 <p>Enter email addresses separated by commas or spaces.</p>
 
 [[inviteUsers]]
 
 <p>Want to learn about other features? Have questions?</p>
-<p><a href="https://docs.wavefront.com/tutorial_overview.html"><img src="images/tut_snag.png"></img></a>&nbsp; &nbsp;
-<a href="https://help.wavefront.com/hc/en-us/requests/new"> <img src="images/get_help_snag.png"></img></a>&nbsp; &nbsp;
+<p><a href="https://docs.wavefront.com/tutorial_overview.html" aria-label="Tutorial and videos"><img src="images/tut_snag.png" alt="Tutorial and videos"></img></a>&nbsp; &nbsp;
```

Comment:
`aria-label="Tutorial and videos"` -> `aria-label="Tutorials and videos"`
`alt="Tutorial and videos"` -> `alt="Video icon"` (and consistency with onboarding_v2/welcome.md)